compiler/rustc_mir/src/transform/qualify_min_const_fn.rs

```diff
@@ -407,6 +407,10 @@ fn check_terminator(
         } => {
             let fn_ty = func.ty(body, tcx);
             if let ty::FnDef(fn_def_id, _) = *fn_ty.kind() {
+                // Panic functions (with one argument) might be const fn.
+                if super::check_consts::is_lang_panic_fn(tcx, fn_def_id) {
+                    return Ok(());
+                }
```

Comment:
With such a change, the most important test to add is to make sure that *without* `const_panic`, the call is still rejected. We likely have such a feature gate test already, but could you make sure?

Reply:
Yes, we have `feature-gate-const_panic.rs` I'll add some more test cases to that file (as it only checks `const`s not `const fn`)

Reply:
Additional checks added.